src/mocks/data/user-group.data.ts

```diff
@@ -50,6 +50,13 @@ export const data: Array<UserGroupResponseModel> = [
 		documentStartNodeId: 'all-property-editors-document-id',
 		permissions: [UMB_USER_PERMISSION_DOCUMENT_CREATE, UMB_USER_PERMISSION_DOCUMENT_DELETE],
 	},
+	{
```

Comment:
Is this user group used for anything? maybe it is just leftover test data?

Reply:
Yea it might be, I cant remember if it was used for anything.